lib/ansible/modules/identity/ipa/ipa_role.py

```diff
@@ -57,6 +57,13 @@
     - If option is omitted host groups will not be checked or changed.
     - If option is passed all assigned hostgroups that are not passed will be unassigned from the role.
     required: false
+  privilege:
+    description:
+    - List of privileges granted to the role.
+    - If an empty list is passed all assigned privileges will be removed.
+    - If option is omitted privileges will not be checked or changed.
+    - If option is passed all assigned privileges that are not passed will be removed.
+    required: false
```

Comment:
When required is False, the option should have a documented default value.  I think None in this case?